src/borg/testsuite/archiver/corruption.py

```diff
@@ -49,10 +51,17 @@ def test_cache_chunks(archiver):
     chunks_path = os.path.join(archiver.cache_path, "chunks")
     chunks_before_corruption = set(ChunkIndex(path=chunks_path).iteritems())
     corrupt(chunks_path)
-    if archiver.FORK_DEFAULT:
-        out = cmd(archiver, "rinfo")
-    else:
+
+    chunks_in_memory = None
+    sync_chunks = LocalCache.sync
+    def sync_wrapper(*args):
```

Comment:
there is only 1 method arg: `self`

so maybe just call it `cache` here?

Reply:
ok